### PR TITLE
- service detail table generator moved to init.py

### PR DESCRIPTION
**Pull request by Anonycoders**
- pending service deploy state added.

### CODE REVIEW

File: fandogh_cli/service_commands.py [hunk 1]
```diff
@@ -196,7 +196,27 @@ def service_apply(file, parameters):
...
 
-    click.echo(message)
+    if detach:
+        click.echo(message)
+    else:
+        while True:
+            details = get_details(service_name)
+
+            if not details:
+                exit(1)
```
Comment (MahdiZareie) [inline fandogh_cli/service_commands.py:207]: why did you exit() here?what does it mean when get_details() returns None?

File: fandogh_cli/service_commands.py [hunk 2]
```diff
@@ -196,7 +196,27 @@ def service_apply(file, parameters):
...
+            if details.get('state') == 'RUNNING':
+                present_service_detail(details)
+                click.echo(message)
+                exit(1)
+            elif details.get('state') == 'UNSTABLE':
+                present_service_detail(details)
+                click.echo('You can press ctrl + C to exit details service state monitoring')
+                sleep(3)
+            else:
+                exit(1)
```
Comment (MahdiZareie) [inline fandogh_cli/service_commands.py:220]: What does it mean when state is neither `RUNNING` nor `UNSTABLE`?

File: fandogh_cli/service_commands.py [hunk 3]
```diff
@@ -196,7 +196,27 @@ def service_apply(file, parameters):
...
+
+            if not details:
+                exit(1)
+
+            click.clear()
+
+            if details.get('state') == 'RUNNING':
+                present_service_detail(details)
+                click.echo(message)
+                exit(1)
```
Comment (MahdiZareie) [inline fandogh_cli/service_commands.py]: use exit(0) :grin:

File: fandogh_cli/service_commands.py (hunk 2 above)
Comment (Anonycoders) [reply fandogh_cli/service_commands.py:220]: it means nothing yet cause we have to solid document yet for states

File: fandogh_cli/service_commands.py (hunk 1 above)
Comment (Anonycoders) [reply fandogh_cli/service_commands.py:207]: Refer to other command ;)

File: fandogh_cli/service_commands.py (hunk 3 above)
Comment (Anonycoders) [reply fandogh_cli/service_commands.py]: agreed

**Pull request merged**